x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/compare_fields_table/get_incompatible_mappings_table_columns/index.test.tsx

```diff
@@ -65,19 +65,6 @@ describe('getIncompatibleMappingsTableColumns', () => {
 
       expect(screen.getByTestId('codeSuccess')).toHaveTextContent(expected);
     });
-
-    test('it renders an empty placeholder when type is undefined', () => {
-      const columns = getIncompatibleMappingsTableColumns();
-      const typeColumnRender = columns[1].render;
-
-      render(
-        <TestProviders>
-          {typeColumnRender != null && typeColumnRender(undefined, eventCategory)}
-        </TestProviders>
-      );
-
-      expect(screen.getByTestId('codeSuccess')).toHaveTextContent(EMPTY_PLACEHOLDER);
-    });
```

Comment:
For reviewers:

This test was removed because it tests an impossible state. Details:
https://github.com/elastic/kibana/pull/185025/files#r1631423077